src/configurator/validators/offline.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import { yellow } from "chalk"
+const log = require("debug")("phenomic:configurator:offline")
+
+const defaultPattern = [ "**", "!**/*.html", "index.html" ]
+const defaultOfflineConfig = {
+  appcache: true,
+  serviceWorker: true,
+  pattern: defaultPattern,
+}
+
+export default (
+  { config, errors }:
+  { config: PhenomicConfig, errors: Array<string>}
+) => {
+  // Deprecated
+  if (config.appcache) {
+    if (typeof config.offline !== "undefined") {
+      errors.push(
+        "phenomic.appcache option was replaced by phenomic.offline option. " +
+        " You can't define both of them at the same time."
+      )
+    }
+    config.offline = true
+
+    log(yellow(
+      "DEPRECATED: phenomic.appcache option was deprecated " +
+      "and will be removed soon. We updated your configurator to use " +
```

Comment:
> We updated your configurator to use

This message is not good. User might think we updated their package.json.
We should say "We assumed you want to enable offline support, so we used"